### PR TITLE
Update Batch Data plane

### PR DESCRIPTION
**Pull request by bgklein**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by bgklein** [comment]
@lmazuel also for review as had to change storage_preparer. 

Saving async for probably v10(its own release probably) to avoid blocking feature release

File: sdk/batch/azure-batch/azure/batch/models/_paged_models.py [hunk 1]
```diff
@@ -145,8 +129,6 @@ class NodeFilePaged(Paged):
     def __init__(self, *args, **kwargs):
 
         super(NodeFilePaged, self).__init__(*args, **kwargs)
-
```
Comment (xingwu1) [inline sdk/batch/azure-batch/azure/batch/models/_paged_models.py]: Don't remove the lines.

File: sdk/batch/azure-batch/azure/batch/models/_paged_models.py [hunk 2]
```diff
@@ -25,8 +25,6 @@ class ApplicationSummaryPaged(Paged):
     def __init__(self, *args, **kwargs):
 
         super(ApplicationSummaryPaged, self).__init__(*args, **kwargs)
-
```
Comment (xingwu1) [inline sdk/batch/azure-batch/azure/batch/models/_paged_models.py]: Don't remove the lines.

**Pull request merged**